index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="style.css">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/all.min.css">
+        <title>YouTube Clone</title>
+    </head>
+<body>
+
+    <header>
+        <div id="brand">
+            <i class="fa-brands fa-youtube"></i>
+            <a href="index.html" class="youtube"><b>YouTube</b></a>
+        </div>
+        <nav>
+            <form class='form-inline'>
+                <div class="input-group">
+                    <input type='text' id='search' class="form-control search-form" placeholder="Search"> 
+                    <span class="input-group-btn"> 
+                    <button id="search-click"type="button" class="pull-right btn btn-default search-btn">
+                    <i class="fa-light fa-magnifying-glass"></i></button>
+                    </span>
+                </div>
+            </form>
+<!-- 
+            <ul class="icons">
+                <li></li>
+                <li></li>
+                <li></li>
+            </ul> -->
+        </nav>
+     </header>
+
+     <main>
+        <section id="video-player">
+            <div class="main-video">
+                <video width="820" height="500" controls>
+                    <source src="video/_A glass of water_ An inspirational story.mp4" type="video/mp4">
```

Comment:
Great Job about adding the video successfully! 
But I wonder if having an online link for the image will not be an easier approach(should it happen that you have multiple videos to showcase in your page"
```suggestion
                    <source src="https://link to youtube">
```